requirements/mint.yml

```diff
@@ -4,14 +4,15 @@ channels:
   - conda-forge
 
 dependencies:
-  - python=3.9
+  - python=3.8  # vtk for py39 is not available from PyPI @March 2021
```

Comment:
@pletzer We're limited to `py38` for now, as we're building `mint` using PEP517/518, we therefore require to use the `vtk` package from PyPI and that's limited to `py38` (latest) at the moment